### PR TITLE
[com_contact] Coding standards form xml + correction wrong closing tags

### PR DESCRIPTION
**Pull request by cyrez**
Code standards (https://joomla.github.io/coding-standards/?coding-standards/chapters/xml.md) + correction closing tags, form xml files com_contact
#### Summary of Changes

**config.xml**
- Code standards
- Add emtpy line before each opening tag (readability)
- Fix wrong field closing tag : <code>image</code>

**models/forms/contact.xml**
- Code standards
- Add emtpy line before each opening tag (readability)
- Fix wrong field closing tag : <code>tags</code>
- Fix mix space/tabs for a few indents

**models/forms/filter_contacts.xml**
- Code standards
- Add emtpy line before each opening tag (readability)
#### Testing Instructions
- code review
- apply and test all is working as before

### CODE REVIEW

File: administrator/components/com_contact/config.xml [hunk 1]
```diff
@@ -43,222 +44,247 @@
...
-			default="10"
 			label="COM_CONTACT_FIELD_ARTICLES_DISPLAY_NUM_LABEL"
 			description="COM_CONTACT_FIELD_ARTICLES_DISPLAY_NUM_DESC"
+			default="10"
 			showon="show_articles:1"
-		>
-			<option value="use_contact">COM_CONTACT_FIELD_VALUE_USE_CONTACT_SETTINGS</option>
+			>
+			<option value="use_contact">
+				COM_CONTACT_FIELD_VALUE_USE_CONTACT_SETTINGS
```
Comment (andrepereiradasilva) [inline administrator/components/com_contact/config.xml]: this line is not 150 characters long so... why break it.

File: administrator/components/com_contact/models/forms/filter_contacts.xml [hunk 2]
```diff
@@ -95,20 +105,31 @@
 			<option value="a.featured DESC">JFEATURED_DESC</option>
 			<option value="access_level ASC">JGRID_HEADING_ACCESS_ASC</option>
 			<option value="access_level DESC">JGRID_HEADING_ACCESS_DESC</option>
-			<option value="association ASC" requires="associations">JASSOCIATIONS_ASC</option>
-			<option value="association DESC" requires="associations">JASSOCIATIONS_DESC</option>
+			<option
+				value="association ASC"
+				requires="associations"
```
Comment (andrepereiradasilva) [inline administrator/components/com_contact/models/forms/filter_contacts.xml]: this line is not 150 characters long so... why break it.

File: administrator/components/com_contact/config.xml (hunk 1 above)
Comment (JoomliC) [inline administrator/components/com_contact/config.xml]: In coding standards for xml, it is said "A max line length of 100 characters is recommended for good reading" ;-)
So, follow the docs here: https://joomla.github.io/coding-standards/?coding-standards/chapters/xml.md
As i've understood it:
- for <code>option</code> tag, when more than one attribute, new line for each attribute (easier to view when not only value attribute)
- when too long text, new line for label.

In the same time, i have set my editor with a max 150 (including indentation). And this line was just 3 characters longer than the full 150 characters width.
That's the reason i use there return to new line ;-)

File: administrator/components/com_contact/models/forms/filter_contacts.xml (hunk 2 above)
Comment (JoomliC) [inline administrator/components/com_contact/models/forms/filter_contacts.xml]: As said in previous comment, the reason is the 2 attributes on the option tag : https://joomla.github.io/coding-standards/?coding-standards/chapters/xml.md

With break, you view faster that not only the value attribute.
Seems to be the standards, and in my opinion it's easier to view the <code>requires</code> attribute ;-)

Comment (andrepereiradasilva) [inline administrator/components/com_contact/models/forms/filter_contacts.xml]: > When the element only has few attributes, then the whole element can stay on the same line. A max line length of 100 characters is recommended for good reading.
> [...]
> Element with only a few attributes
> `<fieldset name="params" label="Some label for the field set">`

Source: https://joomla.github.io/coding-standards/?coding-standards/chapters/xml.md

Comment (JoomliC) [inline administrator/components/com_contact/models/forms/filter_contacts.xml]: Ok about fieldset, but not for option : 
```
    <option
        value="1"
        foo="bar"
        >
```
Source: https://joomla.github.io/coding-standards/?coding-standards/chapters/xml.md

In 99% cases, <code>option</code> tag has only <code>value</code> attribute.
So, don't you think good to view faster when another attribute is set for one option tag ?

Comment (andrepereiradasilva) [inline administrator/components/com_contact/models/forms/filter_contacts.xml]: IMHO, no.

Comment (wojsmol) [inline administrator/components/com_contact/models/forms/filter_contacts.xml]: @JoomliC see https://github.com/joomla/joomla-cms/pull/10482/files/62f485a07949d5c4d0754cde7241f572dce574a2#r63296059

Comment (JoomliC) [inline administrator/components/com_contact/models/forms/filter_contacts.xml]: I will let @Bakual (if possible ;-) ) to comment on this one: the case an option tag has 2 or more attributes.

IMO: the readability is the main concern here, and to quickly visualize extra attribute(s) for an option tag seems to me to be a logical decision... And seems to be the logic of the docs, even if not textualy specified. ;-)

Comment (JoomliC) [inline administrator/components/com_contact/models/forms/filter_contacts.xml]: @wojsmol Nothing related to the mentioned comment ;-) (here, the question is concerning when 2 or more arttibutes for an option tag, not max lenght of an element! ;-) )

File: administrator/components/com_contact/config.xml (hunk 1 above)
Comment (JoomliC) [inline administrator/components/com_contact/config.xml]: @andrepereiradasilva In fact, you're right on this one, does not give any benefit in readability, so just edited as you suggested ;-)
Thanks!

File: administrator/components/com_contact/models/forms/filter_contacts.xml (hunk 2 above)
Comment (Bakual) [inline administrator/components/com_contact/models/forms/filter_contacts.xml]: When an option has attributes beside the value, I would use new lines regardless of length.

Comment (andrepereiradasilva) [inline administrator/components/com_contact/models/forms/filter_contacts.xml]: i don't like it as said, but more "votes" for that, so ... ok

**Comment by andrepereiradasilva** [comment]
I have tested this item :white_check_mark: successfully on 5e84871b7696ad28f5c5426f922655ff11170bcf<br /><br />on code review<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/10504">issues.joomla.org/joomla-cms/10504</a>.</sub>

File: administrator/components/com_contact/models/forms/filter_contacts.xml (hunk 2 above)
Comment (JoomliC) [inline administrator/components/com_contact/models/forms/filter_contacts.xml]: > When an option has attributes beside the value, I would use new lines regardless of length.

Thanks @Bakual for your feedback and advice! 👍

**Comment by freshweb** [comment]
I have tested this item :white_check_mark: successfully on 5e84871b7696ad28f5c5426f922655ff11170bcf<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/10504">issues.joomla.org/joomla-cms/10504</a>.</sub>

**Comment by brianteeman** [comment]
RTC<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/10504">issues.joomla.org/joomla-cms/10504</a>.</sub>

**Comment by roland-d** [comment]
Thanks everybody<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/10504">issues.joomla.org/joomla-cms/10504</a>.</sub>

**Pull request merged**